Foundation/NSTimeZone.swift

```diff
@@ -197,8 +197,19 @@ extension NSTimeZone {
     
     public class func knownTimeZoneNames() -> [String] { NSUnimplemented() }
     
-    public class func abbreviationDictionary() -> [String : String] { NSUnimplemented() }
-    public class func setAbbreviationDictionary(dict: [String : String]) { NSUnimplemented() }
+    public class func abbreviationDictionary() -> [String : String] {
+        let dictionary = CFTimeZoneCopyAbbreviationDictionary()._nsObject
+        var abbreviations = [String : String]()
+        for (key, value) in dictionary {
+            abbreviations[(key as! CFString)._swiftObject] = (value as! CFString)._swiftObject
+        }
+        return abbreviations
+    }
+    
+    public class func setAbbreviationDictionary(dict: [String : String]) {
+        let dictionary: CFDictionary = dict._cfObject
+        CFTimeZoneSetAbbreviationDictionary(dictionary)
```

Comment:
The approach in the PR is to reuse the CFTimeZone abbreviation dictionary - so the get/set updates that and relies of CFs locking.

I can certainly switch to creating a local singleton (initialized from the CF dictionary?) and then maintain that. The side effect is that the NS and CF abbreviation dictionaries can become out of sync. Is that an acceptable side effect?